Formula/odin.rb

```diff
@@ -26,10 +26,9 @@ def install
     llvm = deps.map(&:to_formula).find { |f| f.name.match?(/^llvm(@\d+(\.\d+)*)?$/) }
 
     # Keep version number consistent and reproducible for tagged releases.
-    # Issue ref: https://github.com/odin-lang/Odin/issues/1772
-    inreplace "build_odin.sh", "dev-$(date +\"%Y-%m\")", "dev-#{version}" unless build.head?
-
-    system "make", "release"
+    args = []
+    args << "ODIN_VERSION=dev-#{version}" unless build.head?
```

Comment:
This is needed. Otherwise, the version number would still be obtained from `date +"%Y-%m"`. See: https://github.com/odin-lang/Odin/pull/1876

Reply:
👍